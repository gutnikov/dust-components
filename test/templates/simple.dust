<div>
	{@Button className="simple-button" text="I'm a button"/}
</div>