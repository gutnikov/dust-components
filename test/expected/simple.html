<div>
    <button class="button simple-button">I'm a button</button>
</div>
